docs/api/web-contents.md

```diff
@@ -886,6 +886,33 @@ win.webContents.on('paint', (event, dirty, image) => {
 win.loadURL('https://github.com')
 ```
 
+When using shared texture (set `webPreferences.offscreenUseSharedTexture` to `true`) feature, you can pass the texture handle to external rendering pipeline without the overhead of
+copying data between CPU and GPU memory, with Chromium's hardware acceleration support. This feature is helpful for high-performance rendering scenarios.
+
+It is important to call `texture.release()` when the texture is no longer needed as soon as possible, before the underlying frame pool is drained.
```

Comment:
I'm unfamiliar with the underlying concepts here. Is there a general rule of thumb on the "before the underlying frame pool is drained" part?

And would it be possible to explain what happens if we don't release the texture before the frame pool is drained? (e.g. Will the app crash?)

Reply:
The underlying chromium use FrameSinkVideoCapturer for this feature, and it use RenderableGpuMemoryBufferVideoFramePool to allocate frames. Last time I check it has a const size around 10.

If the buffer is drained the app won't crash, but no more paint events will be populated, unless release is called.